db/migrate/20170421193150_add_embedded_methods_to_miq_ae_method.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmbeddedMethodsToMiqAeMethod < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_ae_methods, :embedded_methods, :text, :array => true
```

Comment:
@mkanoor It's my understanding that we avoid using postgres defaults for all columns and leave all of the default logic in ruby.  So, you can add a line to this migration to populate the column on all existing records.